docs/sources/write/links/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Links
+description: Understand how to link between pages.
+weight: 600
+aliases:
+  - /docs/writers-toolkit/write/links/
+  - /docs/writers-toolkit/write/references/
+  - /docs/writers-toolkit/writing-guide/references/
+keywords:
+  - Hugo
+  - link
+  - linking
+  - links
+  - ref
+  - references
+  - relref
+---
+
+# Links
+
+Choose your link type based on the applicable scenario:
+
+- [Linking from source content that's used (or mounted) in multiple projects](#source-content-is-reused-in-multiple-projects)
+- [Linking to grafana.com pages](#destination-page-is-on-grafanacom)
+- [Linking to external pages](#destination-page-is-external)
+- [Linking to page headings](#anchors)
```

Comment:
Having not looked at this PR for a while and returning to it, I realized having the bullet list of links and the actual headings they link to not have the same text was actually a bit confusing. Suggest we change headings to match link text. 

```suggestion
- [Linking from source content that's used (or mounted) in multiple projects](#linking-from-source-content-thats-used-in-multiple-projects)
- [Linking to grafana.com pages](#linking-to-grafana.com-pages)
- [Linking to external pages](#linking-to-external-pages)
- [Linking to page headings](#linking-to page-headings)
```